cdap-gateway/src/main/java/co/cask/cdap/gateway/router/handlers/SecurityAuthenticationHttpHandler.java

```diff
@@ -80,6 +83,23 @@ public SecurityAuthenticationHttpHandler(String realm, TokenValidator tokenValid
     this.discoveryServiceClient = discoveryServiceClient;
     this.discoverables = discoveryServiceClient.discover(Constants.Service.EXTERNAL_AUTHENTICATION);
     this.configuration = configuration;
+    this.bypassPattern = createMatcher(configuration.get(Constants.Security.Router.BYPASS_AUTHENTICATION_REGEX));
```

Comment:
Isn't `BYPASS_AUTHENTICATION_REGEX` a parameter that is somewhat as sensitive as a password, since allowing people other than the CDAP user to modify this effectively allows them to bypass the authentication mechanism in router?